dev_guide/builds.adoc

```diff
@@ -71,7 +71,8 @@ apiVersion: "v1"
 metadata:
   name: "ruby-sample-build" <1>
 spec:
-  triggers: <2>
+  runPolicy: Serial <2>
```

Comment:
@php-coder @mfojtik yes, to remain consistent with the other areas, quotes should be added to the value.

Also, looks like we'll need a rebase. Thanks!